src/Animation.ts

```diff
@@ -0,0 +1,105 @@
+import {AnimationFrame} from "./AnimationFrame";
+import {ChromaInstance} from "./ChromaInstance";
+import Color from "./Color";
+import {DeviceRequestData} from "./DeviceRequestData";
+import DeviceContainer from "./Devices";
+import { IDevice, IDeviceData } from "./Devices/Base";
```

Comment:
Yes the linting works. :-)
I even made sure that it's called for all subfolders as it didn't before.

It's two different types of import depending if the element imported is a default export or not.
We might however think about not using default exports in general to make it more consistent.
Linting does only care about order of elements. So reordering this to have imports and default imports grouped will cause a linting error cause names aren't ordered alphabetically.